node-proxy/openshift-origin-node-proxy.spec

```diff
@@ -132,6 +132,7 @@ fi
 %endif
 %attr(0755,-,-) %{_bindir}/node-find-proxy-route-files
 %attr(0640,-,-) %{_sysconfdir}/openshift/web-proxy-config.json
+%config(noreplace) %{_sysconfdir}/openshift/web-proxy-config.json
```

Comment:
Actually, if anything, it might make sense to open a Bugzilla report for the fact that `web-proxy-config.json` was not marked as a configuration file.  But probably administrators won't care that much.